src/concept-coach/base.cjsx

```diff
@@ -27,16 +30,29 @@ ConceptCoach = React.createClass
     collectionUUID: React.PropTypes.string.isRequired
 
   getDefaultProps: ->
-    defaultView: 'task'
+    defaultView: _.chain(VIEWS).last().first().value()
```

Comment:
I'm not a big fan of default props because it is unclear to me at least, what is happening when reading where this component is used. For example, it's unclear to me that `<ConceptCoach moduleUUID="123" collectionUUID="234"/>` will show a `task` view instead of the `dashboard` page.